pkg/providers/aws/cluster_network_provider_test.go

```diff
@@ -32,6 +34,7 @@ const (
 	validCIDRTwentyThree       = "10.0.50.0/23"
 	defaultValidSubnetMaskTwoA = "10.0.50.0/24"
 	defaultValidSubnetMaskTwoB = "10.0.51.0/24"
+	defaultNonOverlappingCidr  = "192.0.0.0/20"
```

Comment:
I think we're going to have to take a look at cleaning up some testing in a separate PR. I understand this variable naming now, but I can imagine future readers asking `non overlapping with what?/which cidr?`

I don't think it's important for this though, just a note for myself